recipes/xrandr.lua

```diff
@@ -131,7 +130,6 @@ local function xrandr()
                               function()
                                  state.timer:stop()
                                  state.timer = nil
-                                 state.iterator = nil
```

Comment:
Okay, so restarting the iteration from the beginning is the best idea (in case of monitor disconnection). I'll work it and commit.

I'll wait you to test it so. As we are using the devil's `io.popen`, it's harmless when using 2 monitors. I'd be nice if someone with a big number of monitors tests that module to check whether it works without hanging.